index.html

```diff
@@ -0,0 +1,103 @@
+        literally brooklyn gentrify, roof party seitan direct trade pickled cronut flexitarian. Try-hard knausgaard
+        biodiesel keffiyeh, 90's yr flexitarian man braid pickled messenger bag PBR&B asymmetrical. Listicle vegan XOXO
+        ugh direct trade, single-origin coffee master cleanse scenester. Lyft neutra godard normcore, swag tacos trust
+        und meh pitchfork dreamcatcher pork belly.
+      </p>
+      <p>
+        Offal bitters vaporware yr small batch lomo. Narwhal put a bird on it next level, sriracha lyft fanny pack
+        health goth hashtag bespoke church-key austin photo booth. Keytar activated charcoal pour-over, brooklyn kogi
+        taxidermy vinyl fingerstache. Mumblecore pug craft beer, cliche typewriter leggings fixie pitchfork irony swag.
+        Poke pabst PBR&B, paleo distillery neutra schlitz. Pop-up mustache 8-bit authentic fashion axe, neutra marfa
+        blue bottle kickstarter health goth. Organic health goth lyft, gochujang lomo small batch fingerstache marfa hot
+        chicken.
+      </p>
+      <img src="assets/about-img.jpg" alt="a woman walking through a dessert">
+    </section>
+    <section>
+      <h2>gallery</h2>
+      <p>
+        Hoodie leggings offal, heirloom lo-fi pug freegan put a bird on it hella crucifix authentic beard biodiesel
+        letterpress salvia. Man bun direct trade PBR&B small batch biodiesel meditation, roof party letterpress beard
+        tumblr.
+      </p>
+    </section>
+    <section>
+      <h3>
+        <span>we're open every day</span>
+        <span>from8am - 7pm</span>
+      </h3>
+    </section>
+    <section>
+      <h2>services</h2>
+      <ul>
+        <li>design</li>
+        <li>strategy</li>
+        <li>protyping</li>
+        <li>research</li>
+      </ul>
+      <p>
+        Hoodie leggings offal, heirloom lo-fi pug freegan put a bird on it hella crucifix authentic beard biodiesel
+        letterpress salvia. Man bun direct trade PBR&B small batch biodiesel meditation, roof party letterpress beard
+        tumblr. Air plant shabby chic helvetica post-ironic messenger bag kickstarter.
+      </p>
+      <p>
+        Fam crucifix offal, vaporware blue bottle hashtag brunch. Succulents ennui synth letterpress. Organic next level
+        freegan, viral lumbersexual ennui kinfolk. Twee fixie bitters pop-up, roof party deep v 3 wolf moon irony etsy
+        man braid fap jean shorts.
+      </p>
+      <img src="assets/img.jpg" alt="Hello Studio Office">
+    </section>
+    <section>
+      <h2>contact</h2>
+      <h3>say hello.</h3>
+      <p>
+        Raclette cronut PBR&B, hoodie heirloom blog ethical migas lyft squid jean shorts pinterest succulents fixie
+        synth. Literally skateboard 3 wolf moon organic shabby chic sartorial hell of mumblecore tbh humblebrag. Swag
+        XOXO brunch humblebrag.
+      </p>
+      <form>
```

Comment:
I think this form should go before the explanation. I think it makes enough sense with the heading `contact` and we don't need to change order here.

Reply:
All right. I've had doubts about it, so thanks for pointing it out.